Sentry.EntityFramework note for require ef update (#4502)
getsentry/sentry-docs: src/platforms/dotnet/migration.mdx

Upgrading includes both breaking changes and new features.

## Migrating to 3.12.0

### NuGet Changes

> If using the `Sentry.EntityFramework` NuGet package, an update to at least version 6.2 of the `EntityFramework` NuGet package will also be required.

Reviewer: ```suggestion
If you're using the `Sentry.EntityFramework` NuGet package, you must update to at least version 6.2 of the `EntityFramework` NuGet package.
```
